packages/salesforcedx-vscode-apex-replay-debugger/.vscodeignore

```diff
@@ -7,32 +7,6 @@ src/**
 .gitignore
 tsconfig.json
 
```

Comment:
These vscodeignore files were pretty messy.  By moving to the extensions directory for packaging they are largely unnecessary, but are still providing value by excluding some files from files that get installed in many extensions.